crates/portalnet/src/overlay/service.rs

```diff
@@ -2369,9 +2358,27 @@ impl<
         target: TContentKey,
         callback: Option<oneshot::Sender<RecursiveFindContentResult>>,
```

Comment:
```suggestion
        callback: oneshot::Sender<RecursiveFindContentResult>,
```

I don't think this should be an option